erpnext/erpnext_integrations/doctype/gocardless_settings/gocardless_settings.py

```diff
@@ -85,7 +85,7 @@ def get_payment_url(self, **kwargs):
 		return get_url("./integrations/gocardless_checkout?{0}".format(urlencode(kwargs)))
 
 	def create_payment_request(self, data):
-		self.data = frappe._dict(data)
```

Comment:
Why not frappe._dict.

Reply:
I have updated bench to use python 3  and the data is not formatted correctly with frappe._dict()

Here is my dict with the new method:
```
{'reference_doctype': 'Payment Request', 'description': 'Payment Request for SINV-00043', 'title': 'Test Company', 'order_id': 'PR00016', 'payer_name': 'Test Customer', 'currency': 'EUR', 'payer_email': 'hello@dokos.io', 'amount': '150.0', 'mandate': 'MD0003PSXQVFQD', 'reference_docname': 'PR00016'}
```

With frappe._dict(), I have the following dict and error:
```
{'amount': 300.0, 'payer_name': 'Test Customer', 'description': b'Payment Request for SINV-00033', 'order_id': 'PR00080', 'reference_docname': 'PR00080', 'reference_doctype': 'Payment Request', 'currency': 'EUR', 'title': b'Test', 'mandate': 'MD0003PCY4252C', 'payer_email': 'hello@dokos.io'}
```

```
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/erpnext/erpnext/erpnext_integrations/doctype/gocardless_settings/gocardless_settings.py", line 92, in create_payment_request
    self.integration_request = create_request_log(self.data, "Host", "GoCardless")
  File "/home/frappe/frappe-bench/apps/frappe/frappe/integrations/utils.py", line 62, in create_request_log
    "data": json.dumps(data)
  File "/usr/lib/python3.5/json/__init__.py", line 230, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.5/json/encoder.py", line 198, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.5/json/encoder.py", line 256, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.5/json/encoder.py", line 179, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: b'Test' is not JSON serializable
```

Reply:
Please create an issue on GitHub for this. 

For now, will go ahead with normal dict with `.get()`

Reply:
Hi @saurabh6790,

I have created the following issue https://github.com/frappe/frappe/issues/5854

I have also modified back the method to use the normal dict.

Thanks again for your feedback!